experimental/bridge/opentracing/wrapper.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opentracing
+
+import (
+	"context"
+
+	otelcore "go.opentelemetry.io/api/core"
+	oteltrace "go.opentelemetry.io/api/trace"
+
+	migration "go.opentelemetry.io/experimental/bridge/opentracing/migration"
+)
+
+// WrapperTracer is a wrapper around an OpenTelemetry tracer. It
+// mostly forwards the calls to the wrapped tracer, but also does some
+// extra steps like setting up a context with the active OpenTracing
+// span.
+//
+// It does not need to be used when the OpenTelemetry tracer is also
+// aware how to operate in environment where OpenTracing API is also
+// used.
```

Comment:
not sure what you mean by "when the OpenTelemetry tracer is also aware how to operate in environment where OpenTracing API is also used"?

Reply:
Perhaps an ascii diagram would help.

Reply:
I reworded the package docs, so hopefully it explains it now. They are in the doc.go of the opentracing bridge. I haven't drawn any diagram though. I would need to think more about what to put into the diagram.

Reply:
The new documentation content is great, I'm resolving this comment.